packages/cli/src/commands/generate/schema-from-database/generate_schema_command.ts

```diff
@@ -74,6 +74,7 @@ export class GenerateSchemaCommand
     );
 
     await this.schemaGenerator.generate({
+      secretName,
       connectionUri: connectionUriSecret.value,
```

Comment:
since we're already fetching the secret value at runtime which is `connectionUri` here, why do we need to pass the `secretName` also in addition to that? Can't the underlying libraries directly use the `connectionUri`?

Reply:
We need the secret name to populate in the output file. Refer to the snapshot in the below link.
https://github.com/aws-amplify/amplify-category-api/blob/2c8ee6b16c1bb2b5dececa70354d7f4fff616497/packages/amplify-graphql-schema-generator/src/__tests__/__snapshots__/generate-ts-data-schema.test.ts.snap#L115

For why we don't want to read the connection uri from the secret name in the library, refer to my comment [here](https://github.com/aws-amplify/amplify-backend/pull/1227#discussion_r1544973748).